templates/master/00-master/two-node-with-fencing/OWNERS

```diff
@@ -0,0 +1,5 @@
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+ # This file just uses aliases defined in OWNERS_ALIASES.
+ 
+ reviewers:
+   - edge-enablement-reviewers
```

Comment:
ignoreable nit: might be nice to add newlines to the ends of all these missing newline files